core/src/main/res/layout/fragment_device.xml

```diff
@@ -14,10 +14,17 @@
    permissions and limitations under the License.
 -->
 
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
+    android:layout_height="150dp"
```

Comment:
Am I correctly reading the activity_dev_menu.xml code to have been designed to have its height vary based on the height of its content (such as this)? 

If so, I think it'd be better to have the outer visible dev menu set to a fixed height so that we don't inadvertently have a child  with a taller height than another and the dev menu grows and shrinks in height as you go from one option to another. But maybe I'm missing something in your code and this is not what's happening.

Reply:
You're correct and that is a good point. I'll set the developer menu to have a consistent height across screens.